### PR TITLE
issue #17 authn docs

### PR DESCRIPTION
**Pull request by oraNod**
#17

### CODE REVIEW

File: documentation/asciidoc/topics/yaml/identities_batch.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+stringData:
+  # Optional user for {brandname} metrics.
+  username: monitor
+  # Corresponding password for the "monitor" user.
+  password: password
+  # The key must be 'identities-batch'.
+  # The content is "user create" commands for the {brandname} CLI.
+  identities-batch: |-
+    user create user1 -p changeme -g admin
+    user create user2 -p changeme -g deployer
```
Comment (oraNod) [inline documentation/asciidoc/topics/yaml/identities_batch.yaml:15]: @ryanemerson Does this look correct? I still get an exception in logs when I add this as a secret and configure the Helm chart to use it. I've double checked the secret has the "identities-batch" key.

Comment (ryanemerson) [reply documentation/asciidoc/topics/yaml/identities_batch.yaml:15]: @oraNod What exception do you get?

Comment (ryanemerson) [reply documentation/asciidoc/topics/yaml/identities_batch.yaml:15]: I know the issue now. If the user is creating the secret themselves then they must initialise the `metrics-users.properties` and `metrics-groups.properties` files in their batch OR remove the "metrics" securityRealm from `deploy.infinispan.server.security.securityRealms`.

I would just make the example 

```yaml
  identities-batch: |-
    user create user1 -p changeme -g admin
    user create user2 -p changeme -g deployer
    user create monitor -p password --users-file metrics-users.properties --groups-file metrics-groups.properties
```

File: documentation/asciidoc/topics/yaml/identities_batch.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: connect-secret
+type: Opaque
+stringData:
+  # The key must be 'identities-batch'.
+  # The content is "user create" commands for the {brandname} CLI.
+  identities-batch: |-
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/yaml/identities_batch.yaml:9]: I think it's worth adding the metrics keys and explaining their purpose in this example so something like:

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: connect-secret
type: Opaque
stringData:
  # The user account that will be used for authentication by the Prometheus ServiceMonitor
  username: monitor
  # The password associated with the 'username' account define above
  password: password
  # The key must be 'identities-batch'.
  # The content is "user create" commands for the {brandname} CLI.
  identities-batch: |-
    user create user1 -p changeme -g admin
    user create user2 -p changeme -g deployer
    user create monitor -p password --users-file metrics-users.properties --groups-file metrics-groups.properties
```

**Comment by ryanemerson** [comment]
Thanks @oraNod

**Pull request merged**